### PR TITLE
Blog on Quarkus JDiameter Extension

### PR DESCRIPTION
**Pull request by eddiecarpenter**
** If you are updating a guide, please submit your pull request to the main repository: https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc **

### CODE REVIEW

File: _data/authors.yaml [hunk 1]
```diff
@@ -559,3 +559,10 @@ karesti:
   job_title: "Engineer (Software)"
   twitter: "karesti"
   bio: "Software Engineer at Red Hat and Infinispan team member."
+eddiecarpenter:
+    name: "Eddie Carpenter"
+    email: "eddie.carpenter@icloud.com"
+    emailhash: "1eafb77e5daee9aba696f7c23de0bc86"
+    job_title: "Prinsiple Software Engineer"
```
Comment (gastaldi) [inline _data/authors.yaml:566]:
```suggestion
    job_title: "Principal Software Engineer"
```

Comment (eddiecarpenter) [reply _data/authors.yaml:566]: 100% - That one slip through ☺️

File: _posts/2024-08-19-quarkus-jdiameter-intro.adoc [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+The next challenge was to develop a Quarkus extension to integrate the Diameter stack into the Quarkus framework. I had three main objectives with this extension:
+
+. *Simplified Configuration*: Moving the configuration to the "application.properties“ file to make the stack easier to configure.
+. *Dependency Injection*: Enabling dependency injection of Diameter stacks and configuration into a scoped service.
+. *Interceptor Service Framework*: Creating an interceptor service framework for a more straightforward implementation of Diameter applications.
+
+The Quarkus JDiameter stack has been released and is now part of Quarkiverse Hub. You can find it at link:https://github.com/quarkiverse/quarkus-jdiameter[Quarkiverse Hub]. Documentation for the stack is available at link:https://docs.quarkiverse.io/quarkus-jdiameter/2.0.0/index.html[Quarkus JDiameter Documentation].
```
Comment (gastaldi) [inline _posts/2024-08-19-quarkus-jdiameter-intro.adoc:38]:
```suggestion
The Quarkus JDiameter stack has been released and is now part of Quarkiverse Hub. You can find it at link:https://github.com/quarkiverse/quarkus-jdiameter[Quarkiverse Hub]. Documentation for the stack is available at link:https://docs.quarkiverse.io/quarkus-jdiameter/dev/index.html[Quarkus JDiameter Documentation].
```

You may want to point the URL to the main dev branch

Comment (eddiecarpenter) [reply _posts/2024-08-19-quarkus-jdiameter-intro.adoc:38]: To clarify, it should point to 2.0.0

File: _posts/2024-08-19-quarkus-jdiameter-intro.adoc [hunk 3]
```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: 'Revolutionsing Telecom Microservice - Modernising JDiameter with Quarkus'
```
Comment (gastaldi) [inline _posts/2024-08-19-quarkus-jdiameter-intro.adoc:3]:
```suggestion
title: 'Revolutionising Telecom Microservice - Modernising JDiameter with Quarkus'
```

**Pull request merged**